tests/functional/finetune/finetune_functional_test.py

```diff
@@ -95,6 +109,13 @@ def test_end2end(run_input_map):
         assert len(result["details"][0]["metadata"]) > 0
     finetune_model.delete()
 
+def test_end2end(run_input_map, finetunable_llms_from_last_4_weeks):
+    if finetunable_llms_from_last_4_weeks:
+        for model in finetunable_llms_from_last_4_weeks:
+            run_finetune_test(model, run_input_map)
```

Comment:
You should not test all the models within the same method. You should call `test_end2end` for each model in `finetunable_llms_from_last_4_weeks`. Check how to use a fixture together with the python command `yield`. There is an example here: https://github.com/aixplain/aiXplain/blob/main/tests/functional/pipelines/designer_test.py#L17